readme.md

```diff
@@ -42,6 +42,8 @@ On Linux:
 
 For more information see the [detailed instructions](docs/deploy/howto-deploy-all-in-one.md) and [alternative deployment options](docs/deploy/readme.md).
 
+For detailed documentation of Azure Industrial IoT Platform, please refer to [Operations Manual](https://github.com/Azure/Industrial-IoT/tree/main/docs/manual).
+
```

Comment:
Use a markdown relative reference instead of a url.  E.g. docs/manual/readme.md

Reply:
fixed